code/modules/maps/lateload/generic/gateway_lateload.dm

```diff
@@ -0,0 +1,105 @@
+////////////////////////////////////////////////
+///////    Gateway Templates     ///////////////
+////////////////////////////////////////////////
+
+/*
+This is for gateway maps. These are unique in that there should only ever be one loaded in if any. As well you can make them
+overmap accessible or not if you like.
+
+If you are wanting to add a map like lavaland that isn't accessible
+from the overmap and *are not* gateway missions please put it in unique_lateload.dm
+
+For overmap things that are planets and accessible from the overmap please put them
+in planets_lateload.dm
+
+For overmap things that are in space and *are not* planets put them in space_lateload.dm
+
+*/
+
+/datum/map_template/gateway_lateload
```

Comment:
Mostly just making the multi-line comment look and read better, also putting it up to `gateway_lateload` so it can be read by DMDOC
```suggestion
/**
 *! Gateway Templates
 *
 *? This is for gateway maps.
 *? These are unique in that there should only ever be one loaded in if any.
 *? As well you can make them overmap accessible or not if you like.
 *
 *! NOTICE
 *
 *? unique_lateload.dm
 *  * For maps that aren't accessible from overmaps or gateway missions.
 *
 *? planets_lateload.dm
 *  * For plants that are accessible from the overmap.
 *
 *? space_lateload.dm
 *  * For overmap locations that are in space and *are not* planets.
 */
/datum/map_template/gateway_lateload
```